features/cellular/framework/targets/UBLOX/C027/UBLOX_C027.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef UBLOX_C027_H_
+#define UBLOX_C027_H_
+
+#include "AT_CellularDevice.h"
+
+namespace mbed {
+
+class UBLOX_C027 : public AT_CellularDevice
```

Comment:
This seems like a misnomer? The C027 is the board, not the modem.

Indeed, is there even anything C027-specific here? Isn't this just the "generic onboard PPP modem, using onboard power APIs?"

Reply:
As a placeholder, renamed AT module to LISA_U
https://github.com/AriParkkila/mbed-os/commit/7392e73f1714400a726019dc2dc44d57464621bf